ui/src/main/resources/application-local.properties

```diff
@@ -0,0 +1 @@
+logging.level.=DEBUG
```

Comment:
should be removed

Reply:
Maybe I should better also remove it from git, as it is also a local settings file?

Reply:
yes, please

Reply:
Removed